source/RaidFleets.h

```diff
@@ -0,0 +1,42 @@
+/* RaidFleets.h
+Copyright (c) 2023 by Hurleveur
+
+Endless Sky is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free Software
+Foundation, either version 3 of the License, or (at your option) any later version.
+
+Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+this program. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#ifndef RAID_FLEET_H_
+#define RAID_FLEET_H_
+
+class Fleet;
+class DataNode;
+
+#include <vector>
+
+class RaidFleet {
+public:
+	RaidFleet(const Fleet *fleet, double minAttraction, double maxAttraction);
+	const Fleet *GetFleet() const;
+	double MinAttraction() const;
+	double MaxAttraction() const;
+
+private:
+	const Fleet *fleet = nullptr;
+	double minAttraction;
+	double maxAttraction;
+};
+
+class RaidFleets : public std::vector<RaidFleet> {
```

Comment:
If you make Load a static function of RaidFleet you can avoid having to define this type if you want (I'm fine either way).

Reply:
idk I dont mind the inheritance, not sure how you see your alternative tho tbh

Reply:
Like this `RaidFleet::Parse(raidFleets, node, remove, valueIndex);`, and it would take `raidFleets` by reference.

Reply:
oh then we dont need RaidFleets at all as an abstraction
what you're saying feels wrong OO but I dont care bout that and neither does the code in general so... sure